code/datums/mind.dm

```diff
@@ -774,6 +774,17 @@
 	if(G)
 		G.reenter_corpse()
 
+//user: The mob that is suiciding
+/datum/mind/proc/suicide_log(mob/user)

```

Comment:
Now that I think about it, can I ask what's the purpose of putting this in the mind if you're never actually using the mind?

If you put this on mob/living you can manipulate the proc itself for humans which I think would be slightly more optimized than checking if it's a human anytime a mob suicides.

Reply:
I'll make the change you want, but I'm pretty sure an unnecessary type check on each suicide isn't that big of a performance loss in the grand scheme of things.

Reply:
you're certainly correct, I think from a backseat-maintainer  / "good coding practices" standpoint I'd prefer to have it off mind unless it is used somewhere within the block that i'm missing.

Reply:
You're right about that. I was originally planning to shove the var into the mind when I was making the previous PR, but decided against that and somehow thought it would be a great place for this proc.

Reply:
All is well! Just wanted to make sure I wasn't missing anything!